app/models/payments/OpenPaymentsModel.scala

```diff
@@ -49,6 +49,7 @@ sealed trait OpenPaymentsModel {
     case FailureToSubmitRCSLCharge => messages("openPayments.vatFailureToSubmitRCSL")
     case VatInaccuraciesInECSalesCharge => messages("openPayments.vatInaccuraciesECSales")
     case VatFailureToSubmitECSalesCharge => messages("openPayments.vatFailureToSubmitECSales")
+    case VatPaFurtherInterestCharge => messages("openPayments.vatPaFurtherInterest")
```

Comment:
Could you add a VAT PA Further Interest charge type in the Acceptance test stub data (`user6Open.json`) to make this show up please?